src/status_im2/contexts/communities/home/view.cljs

```diff
@@ -46,20 +46,20 @@
     :render-fn                         render-fn}])
 
 (defn segments-community-lists [selected-tab]
-  (let [communities (rf/sub [:communities/community-ids])
+  (let [communities-ids (rf/sub [:communities/community-ids-by-user-involvement])
         tab @selected-tab]
     [rn/view {:style {:padding-left     20
                       :padding-right    8
                       :padding-vertical 12}}
      (case tab
        :joined
-       [communities-list communities]
+       [communities-list (:joined communities-ids)]
 
        :pending
-       [communities-list communities]
+       [communities-list (:pending communities-ids)]
 
        :opened
-       [communities-list communities])]))
+       [communities-list (:opened communities-ids)])]))
```

Comment:
`case` without a default branch will throw an exception. If you're sure the tab is always within the valid set of branches, another alternative is to remove `case` and just use the following:

```clojure
[communities-list (get communities-ids tab)]
```

Reply:
I would prefer not to rely on using the same keywords for naming tabs and for naming vectors of communities, since they have different logical meanings and can change in the future. So I would add the :default section with simple error indication to visually catch this case. Wdyt?
```
:default
       [quo/information-box
        {:type :error
         :icon :i/info}
        (i18n/label :t/error)]
        ```

Reply:
Makes total sense to me @vkjr. Thanks